src/background/utils/overwriteWitnetNodeConfiguration.ts

```diff
@@ -45,15 +48,10 @@ export function overwriteWitnetNodeConfiguration(
     const currentNodeUrls: string =
       nodeUrlConfigLine.match(nodeUrlsRegex)?.[0] || ''
     const configNodeIps: Array<string> | undefined = currentNodeUrls
+      .replaceAll(' ', '')
       .replaceAll('"', '')
       .split(',')
-    if (
-      configNodeIps &&
-      existIntersection({
-        oldWitnetNodes: oldWitnetNodes,
-        currentNodeConfiguration: configNodeIps,
-      })
-    ) {
+    const overwriteConfig = () => {
```

Comment:
We don't need to extract this logic into a function. We can write it directly inside the `if`